fsevents.js

```diff
@@ -6,7 +6,11 @@
 /* jshint node:true */
 'use strict';
 
-var Native = require('./build/Release/fse');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var Native = require(binding_path);
```

Comment:
Doesn't simply `var Native = require(binary.find(__dirname + '/package.json'))` work?

Style nits: please use camelCase, wrap lines at 80 columns and put a space after commas.